Example/GiniVision.xcodeproj/project.pbxproj

```diff
@@ -360,6 +360,7 @@
 						CreatedOnToolsVersion = 6.3.1;
 						DevelopmentTeam = JA825X8F7Z;
 						LastSwiftMigration = 0810;
+						ProvisioningStyle = Automatic;
```

Comment:
I think it has been generated automatically when I used Gini provisioning profile. Does it need to be manual rather than automatic for non-release purposes?